lib/cronitor.rb

```diff
@@ -57,7 +57,7 @@ def exists?(name)
 
   def ping(type, msg = nil)
     url = "#{PING_URL}/#{code}/#{type}"
-    url += "?msg=#{URI.escape msg}" if type == 'fail' && !msg.nil?
+    url += "?msg=#{URI.escape msg}" if ['run', 'complete', 'fail'].include?(type) && !msg.nil?
```

Comment:
Might as well just remove `['run', 'complete', 'fail'].include?(type)` since that's all the available types anyway